include/interfaces/compressible_neo_hookean_iterative.h

```diff
@@ -0,0 +1,181 @@
+#ifndef _pidoums_compressible_neo_hookean_h_
+#define _pidoums_compressible_neo_hookean_h_
+
+#include "interface.h"
+
+#include <deal.II/lac/linear_operator.h>
+#include <deal.II/lac/block_linear_operator.h>
+#include <deal.II/lac/packaged_operation.h>
+#include <deal.II/lac/solver_cg.h>
+
+
+template <int dim, int spacedim>
+class CompressibleNeoHookeanInterface : public Interface<dim,spacedim,dim, CompressibleNeoHookeanInterface<dim,spacedim> >
+{
+  typedef TrilinosWrappers::MPI::BlockVector BVEC;
+  typedef TrilinosWrappers::MPI::Vector VEC;
+  typedef TrilinosWrappers::BlockSparseMatrix MAT;
```

Comment:
use `LAC::MatrixType` and `LAC::VectorType`. To access subblock vector types: `LAC::VectorType::BlockType`